### ISSUE TITLE
During connections with Snapshot Database

### ISSUE THREAD

**Issue by ijunaid8989** [evercam/evercam-server#727]
This issue is occurring few times, while evercam server trying to make connection with Snapshot Database.
```
** (Postgrex.Error) ssl recv: TLS connection is closed - :closed
            (ecto) lib/ecto/adapters/sql.ex:412: Ecto.Adapters.SQL.sql_call!/6
            (ecto) lib/ecto/adapters/sql.ex:394: Ecto.Adapters.SQL.execute/6
            (ecto) lib/ecto/repo/queryable.ex:127: Ecto.Repo.Queryable.execute/5
            (ecto) lib/ecto/repo/queryable.ex:40: Ecto.Repo.Queryable.all/4
            (ecto) lib/ecto/repo/queryable.ex:64: Ecto.Repo.Queryable.one/4
   (evercam_media) lib/mix/tasks/add_camera_logs.ex:15: anonymous fn/1 in EvercamMedia.AddCameraLogs.run/0
          (elixir) lib/enum.ex:651: Enum."-each/2-lists^foreach/1-0-"/2
          (elixir) lib/enum.ex:651: Enum.each/2
```

Update:

```
[error] Postgrex.Protocol (#PID<0.2347.0>) disconnected: ** (DBConnection.ConnectionError) client #PID<0.8904.84> timed out because it checked out the connection for longer than 60000ms
```

**Comment by ijunaid8989** [comment]
Due to SSL TLS issues the SnapShotRepo is being crashed, and logs are full of such errors

```
{%ArgumentError{message: "repo EvercamMedia.SnapshotRepo is not started, please ensure it is part of your supervision tree"},
 [{Ecto.Query.Planner, :query_lookup, 6,
   [file: 'lib/ecto/query/planner.ex', line: 136]},
  {Ecto.Query.Planner, :query_with_cache, 7,
   [file: 'lib/ecto/query/planner.ex', line: 119]},
  {Ecto.Repo.Queryable, :execute, 5,
   [file: 'lib/ecto/repo/queryable.ex', line: 122]},
  {Ecto.Repo.Queryable, :all, 4,
   [file: 'lib/ecto/repo/queryable.ex', line: 35]},
  {Ecto.Repo.Queryable, :one, 4,
   [file: 'lib/ecto/repo/queryable.ex', line: 68]},
  {EvercamMedia.Snapshot.DBHandler, :check_last_camera_status, 2,
   [file: 'lib/evercam_media/snapshot/db_handler.ex', line: 129]},
  {EvercamMedia.Snapshot.DBHandler, :"-do_log_camera_status/4-fun-0-", 4,
   [file: 'lib/evercam_media/snapshot/db_handler.ex', line: 117]}]}
```

https://service.eu.sumologic.com/ui/#section/search/GUgtOGg3ScTK0oDb9WsZx0GlTToRRMxQ3XzNL6cw

**Comment by ijunaid8989** [comment]
I had no idea what to do or what was going wrong so I posted a question to Elixir forum they pointed me few things and I made changes according to that, while those changes and testing I found out there are many many other issues related to this Online offline thing, 

- Our Repo for SnapShotDB is crashing again and again.

- also the query to DB while searching last log for online or offline, timing out

- and there are many Postrex.Connection close issues, 

The fix is still online, during last 3 days, repo crash are in low  amount but the query is still timing out.

Right now I am deleting data, from the table.

At first..

-  I was just deleting between 2 ranges (which is a lot of time taking)

 But Now..

- [ ] I am just duplicating the whole table

- [ ] Adding the greater than 2016 records into it

- [ ] , after that I will rename this one to camera activities DB

- [ ]  and then drop the first one.
